.github/dependabot.yml

```diff
@@ -25,13 +25,13 @@ updates:
     interval: daily
   open-pull-requests-limit: 10
 - package-ecosystem: bundler
-  target-branch: "inspec-6"
```

Comment:
At this time we are not deleting the InSpec 6 branch - that will happen in a merge.